target_athena/target.py

```diff
@@ -22,6 +22,7 @@ class TargetAthena(Target):
         th.Property("aws_profile", th.StringType),
         th.Property("s3_key_prefix", th.StringType),
         th.Property("naming_convention", th.StringType),
+        th.Property("object_format", th.StringType, default='csv'),
```

Comment:
What do you think of `stage_file_format`, `s3_file_format`, or `file_format` instead of `object_format`. I recognize that there is technically a distinction between 'objects' and 'files' but for most users, I think `file_format` is a little more obvious in its meaning. (Technically they are files before they get uploaded as 'objects' 😄 .)